assets/scss/components/_slide.scss

```diff
@@ -0,0 +1,50 @@
+.slide {
+  min-height: 600px;
+
+  &__image {
+    margin: 0 auto;
+    margin-bottom: 38px;
+  }
+
+  &__title {
+    margin-bottom: 25px;
+    color: $color-text-grey;
+    font-size: 50px;
+    line-height: 61px;
+  }
+
+  &__title-emphasis {
+    display: block;
+    font-weight: bold;
+  }
+
+  &__subtitle {
+    margin-bottom: 30px;
+    color: #9B9B9B;
+    font-size: 30px;
+    line-height: 1;
+  }
+
+  &__button {
+
+  }
+
+}
+
+.slides {
```

Comment:
As weird as this sounds, if you have `.slides`, I would put it in it's own file, since it's inherently different than the `.slide` component.  That being said, if you want to put these in the same file, I would make your parent class `.slides` and then put in `.slides__slide` for each slide.